cvmfs/statistics_database.cc

```diff
@@ -490,10 +523,14 @@ bool StatisticsDatabase::UploadStatistics(
 }
 
 
-std::string StatisticsDatabase::GetDBPath(const std::string &repo_name) {
+void StatisticsDatabase::GetDBParams(const std::string &repo_name,
+                                     std::string *path,
+                                     uint32_t *days_to_keep)
+{
   // default location
   const std::string db_default_path =
       "/var/spool/cvmfs/" + repo_name + "/stats.db";
+  const uint32_t default_days_to_keep = 365;
```

Comment:
Let's make this a private static const in the header of the `StatisticsDatabase` class